Package installation test is failing due to new version of packaging module

The `test_package_installation_with_target_path` test is failing because a new version (3 actually) of the packaging module was released yesterday that moved the version from [20.4 to 20.7](https://github.com/pypa/packaging/releases).  This leads to the following test failure...

```
        for package, version in correct_dict.items():
>           assert virtual_env_dict[package] == version
E           AssertionError: assert '20.7' == '20.4'
E             - 20.4
E             ?    ^
E             + 20.7
E             ?    ^

etc/tests/test_bootstrapper.py:316: AssertionError
```

I'm not certain if the test should still pass - forcing the use of 20.4 or whether a newer version should be honored, in which case, the test's assertion enforcement should be made more flexible.

--- a/etc/tests/test_bootstrapper.py
+++ b/etc/tests/test_bootstrapper.py
@@ -523,14 +523,14 @@ def test_package_installation_with_target_path(monkeypatch, virtualenv):
                   }
     to_install_dict = {'bleach': '3.1.5',
                        'ansiwrap': '0.7.0',
-                       'packaging': '20.4',
+                       'packaging': '20.7',
                        'text-extensions-for-pandas':
                        "git+https://github.com/akchinSTC/"
                        "text-extensions-for-pandas@50d5a1688fb723b5dd8139761830d3419042fee5"
                        }
     correct_dict = {'ipykernel': '5.3.0',
                     'ansiwrap': '0.8.4',
-                    'packaging': '20.4',
+                    'packaging': '20.7',
                     'text-extensions-for-pandas':
                     "git+https://github.com/akchinSTC/text-extensions-for-pandas@50d5a1688fb723b5dd8139761830d3419042fee5"  # noqa: E501
                     }
@@ -542,7 +542,7 @@ def test_package_installation_with_target_path(monkeypatch, virtualenv):
 
     virtualenv.run("python3 -m pip install --target='/tmp/lib/' bleach==3.1.5")
     virtualenv.run("python3 -m pip install --target='/tmp/lib/' ansiwrap==0.7.0")
-    virtualenv.run("python3 -m pip install --target='/tmp/lib/' packaging==20.4")
+    virtualenv.run("python3 -m pip install --target='/tmp/lib/' packaging==20.7")
     virtualenv.run("python3 -m pip install --target='/tmp/lib/' git+https://github.com/akchinSTC/"
                    "text-extensions-for-pandas@50d5a1688fb723b5dd8139761830d3419042fee5")
 

